### PR TITLE
Include gcc C library and utils documentation

### PR DESCRIPTION
**Pull request by cua-cua**

### CODE REVIEW

File: docs/source/utils.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.. index:: utils
+
+
```
Comment (Peque) [inline docs/source/utils.rst]: I would only leave one blank line bellow `.. index::`. :wink:

File: docs/source/utils.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+.. code-block:: bash
+
+   pip install -r requirements.txt
+
+
+.. index::
+    double: utils; documentation
+
+
```
Comment (Peque) [inline docs/source/utils.rst]: I would only leave one blank line bellow `.. index::`. :wink:

File: docs/source/utils.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.. index:: utils
+
+
+*****
+Utils
+*****
+
+.. index::
+    double: utils; dependencies
```
Comment (Peque) [inline docs/source/utils.rst]: Maybe you could simply say: `.. index:: utils, dependencies`

File: docs/source/utils.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+To install the required dependencies for developing the Python code of Theseus, you can
+make use of the provided ``requirements.txt`` file:
+
+.. code-block:: bash
+
+   pip install -r requirements.txt
+
+
+.. index::
+    double: utils; documentation
```
Comment (Peque) [inline docs/source/utils.rst]: Maybe you could use: `.. index:: sphinx, documentation` here?

**Comment by cua-cua** [comment]
I updated style comments. I copied utils from markdownreveal developers file (just in case you want to update that also there).

**Pull request merged**